### PR TITLE
DXA2 Recommendations #12, #27

### PR DESCRIPTION
**Pull request by kimberlyxdavis**
Low/Medium items from DXA2 Recommendations.

### CODE REVIEW

File: docs/General/createCIRAConfig.md [hunk 1]
```diff
@@ -1,38 +1,42 @@
...
+6. Set **User Name** and **Password** to the following:
 
     | Field         | Value      |
     | :------------ | :--------- |
     | **User Name** | standalone |
     | **Password**  | G@ppm0ym   |
 
 
-7. Provide your developement system's IP address as the Common Name to generate the self-signed certificate by the MPS server.
+7. Provide your developement system's IP address as the **Common Name** to generate the self-signed certificate by the MPS server.
```
Comment (rsdmike) [inline docs/General/createCIRAConfig.md:27]: developement -> development

File: docs/General/createCIRAConfig.md [hunk 2]
```diff
@@ -1,38 +1,42 @@
...
 
-8. Leave Mps root certificate format to the default, Auto Load.
+8. Do not modify the default value of **Auto Load** for the **Mps root certificate format.**
```
Comment (rsdmike) [inline docs/General/createCIRAConfig.md:29]: i think MPS should be capitalized here.

File: docs/General/createProfileCCM.md [hunk 3]
```diff
@@ -1,17 +1,36 @@
...
+- [**Client Control Mode (CCM):**](../Glossary.md#c) This mode limits some of Intel AMT functionality, reflecting the lower level of trust.
+
+    Features requiring User Consent:
+
+    - Keyboard, Video, Mouse (KVM) Control
+    - IDE-Redirection for sharing and mounting images remotely
+
+![assets/images/Profiles.png](../assets/images/Profiles.png)
+
+**Figure 1: Set up configuation and profiles for N number of clients. **
```
Comment (rsdmike) [inline docs/General/createProfileCCM.md:14]: configuation -> configuration

**Review by rsdmike — CHANGES_REQUESTED** [review]
just a couple small spelling changes, otherwise looks good

**Comment by kimberlyxdavis** [comment]
Fixed spelling errors.

**Comment by kimberlyxdavis** [comment]
Not sure who merges the files. You?

**Review by rsdmike — APPROVED** [review]

**Pull request merged**